sigs.yaml

```diff
@@ -1062,12 +1062,15 @@ sigs:
       description: Test Failures and Triage
   subprojects:
   - name: community
+    description: Owns and manages overall community repo, including SIG description and charter maintenance.
     owners:
     - https://raw.githubusercontent.com/kubernetes/community/master/OWNERS
   - name: community-management
+    description: Manages all communications mechanisms for the community, including Slack, mailing lists, Zoom, social media, and YouTube.
```

Comment:
Can we be more broad here? We don't own social media like Twitter and SIGs technically own their own lists, we help moderate in certain cases, and own others like k-dev. 

'operations and policy for upstream community group communication platforms'